build-scripts/ubuntu-1804/build-indy-plenum.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -xe
+
+INPUT_PATH=$1
+VERSION=$2
+OUTPUT_PATH=${3:-.}
+PACKAGE_VERSION=${4:-$VERSION}
+
+PACKAGE_NAME=indy-plenum
+
+# copy the sources to a temporary folder
+TMP_DIR=$(mktemp -d)
+cp -r ${INPUT_PATH}/. ${TMP_DIR}
+
+# prepare the sources
+cd ${TMP_DIR}/build-scripts/ubuntu-1804
+./prepare-package.sh ${TMP_DIR} ${VERSION}
+
+sed -i 's/{package_name}/'${PACKAGE_NAME}'/' "postinst"
+sed -i 's/{package_name}/'${PACKAGE_NAME}'/' "prerm"
+
+fpm --input-type "python" \
+    --output-type "deb" \
+    --architecture "amd64" \
+    --verbose \
+    --python-package-name-prefix "python3"\
+    --python-bin "/usr/bin/python3" \
+    --exclude "usr/local/lib/python3.5/dist-packages/data" \
```

Comment:
I think, that this deb should be built in ubuntu 18 environment and using python 3.6. Therefore, path to exclude should be the same as in ubuntu 18 (maybe changing python3.5 to python3.6 will be enough)